schemas/src/elements/Column.json

```diff
@@ -0,0 +1,69 @@
+{
+	"$schema": "https://raw.githubusercontent.com/microsoft/AdaptiveCards/6f39aedce45864ae1067ed44a5551dc973790bb5/source/nodejs/typed-schema/schema/lib/Type.json",
+	"extends": "ToggleableItem",
+	"description": "Defines a container that is part of a ColumnSet.",
+	"properties": {
+		"items": {
+			"type": "Element[]",
+			"description": "The card elements to render inside the `Column`."
+		},
+		"backgroundImage": {
```

Comment:
we may want something like a `ContainerElement` base so we don't have to duplicate `backgroundImage` (and other) entries

Reply:
Could be a good future improvement, as long as ContainerElement doesn't inherit from Element, otherwise you can then put a Column in the body.